### ISSUE TITLE
1.15.2: WorldEdit block limit is ignored

### ISSUE THREAD

**Issue by HgeX** [kangarko/Confiscate#98]
So I have WorldEdit hook enabled in my configuration, and the `Total_Blocks_Limit` works just fine, but the limit for individual blocks, not so much. The plugin gets enabled as expected with no errors whatsoever.

[This](https://hastebin.com/zizikatino.php) is how the config section looks like.
You can access the plugin's folder by clicking [here](https://drive.google.com/file/d/1J7ZyitaehfzV8mSbkPkhAwU1DLaCF6Tk/view?usp=sharing)

According to the configuration, one without confiscate.bypass should be able to place only 32 sponges at a time, however the operation completes every time, no matter how much sponge do I attempt to place. Blocks without any limit set, should be completely removed from any operation, but they are not.
*Basically the whole section is ignored, or I'm doing something seriously wrong... :/*

**Plugin version**: Confiscate 2.4.1
**WorldEdit version**: 7.1.0;8e55131
**Server version**: git-Paper-346 (MC-1.15.2)

**Plugins**:
`
AcceptTheRules, AntiAFKPro, BlockParticles, BuildersUtilities, ChatControl, Citizens, CitizensCMD, Confiscate, DeluxeMenus, FogusCore, HolographicDisplays, LiteChat, LuckPerms, Multiverse-Core, NuggetCore, ParancsKuld-SPIGOT, PlaceholderAPI, PlotSquared, ProtocolLib, ServerSigns, SkinsRestorer, spark, SunLight, SunLight_Economy, UltraBar, Vault, ViaBackwards, ViaRewind, ViaVersion, WorldBorder, WorldEdit, WorldEditSUI, WorldGuard
`

**Comment by JavaFactoryDev** [comment]
Hi HgeX, 
thank you for reporting this issue here, we will have a look at it.

**Comment by kangarko** [comment]
Confirmed, failed to fix, waiting for support: https://www.spigotmc.org/threads/how-to-convert-worldedits-blockstateholder-into-bukkits-material.456946/

**Comment by kangarko** [comment]
Fixed!
